testassets/InteropTestsWebsite/InteropTestsWebsite.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks Condition="'$(LatestFramework)'!='true'">net6.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(LatestFramework)'=='true'">net5.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(LatestFramework)'=='true'">net6.0</TargetFrameworks>
```

Comment:
Is there any point in having these two conditions at all if we are defaulting to `net6.0` anyways?

Reply:
I left it as is because it will change when the SDK and projects update to net7.